### PR TITLE
[Term Entry] JavaScript Dates .setHours()

### PR DESCRIPTION
**Pull request by eskuode**
### Description

I have made an introductory paragraph about .setHours method, what it is and what it does. Also, I made  a section about parameters used in this method. And finally, I have provided an example showing the method in use.

- 

### Type of Change

- Adding a new entry

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:40]:
```suggestion
In the example below, the `myDate` variable takes the provided date and applies the `.setHours()` method to set the hours, minutes, and seconds of the date:
```

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:36]:
```suggestion
**`hoursValue:`** 
- Required. 0 to 23 representing the hours.
**`minutesValue:`**
- Optional. 0 to 59 representing the minutes.
**`secondsValue:`**
- Optional. 0 to 59 representing the seconds. You must also specify `minutesValue`.
**`millisecondsValue:`**
- Optional. 0 to 999 representing the milliseconds. You must also specify `minutesValue` and `secondsValue`.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side. Also please remove that other new_file.html file from this PR.

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:25]:
```suggestion
```

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:27]: `.setHours()` takes the following parameters:

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:30]:
```suggestion
- `hoursValue:` (Required) Takes values from `0` to `23` representing the hours.
```

Comment (caupolicandiaz) [reply content/javascript/concepts/dates/terms/setHours/setHours.md:30]: update the other parameters as above

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:25]:
```suggestion
The `.setHours()` method changes the hours, minutes, seconds and milliseconds for a given date according to local time.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@eskuode I've completed a maintainers review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
@eskuode Are you still working on these changes? It looks as though some updates are still missing.

**Comment by eskuode** [comment]
@caupolicandiaz hi, yes i am still working on it

**Comment by eskuode** [comment]
@caupolicandiaz I will let you know in couple days how it's going, just messed up something on my local repo.

**Comment by eskuode** [comment]
@caupolicandiaz I have finished with updates.

**Review by caupolicandiaz — APPROVED** [review]
@eskuode Overall, looks good. I did push a couple of minor edits. I'm passing this on to a second review.

File: content/javascript/concepts/dates/terms/setHours/setHours.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setHours/setHours.md:30]:
```suggestion
- `hoursValue`: (Required) takes values from `0` to `23` representing the hours.
- `minutesValue`: (Optional) takes values from `0` to `59` representing the minutes.
- `secondsValue`: (Optional) takes values from `0` to `59` representing the seconds. Must also specify `minutesValue`.
- `millisecondsValue`: (Optional) takes values from `0` to `999` representing the milliseconds. Must also specify `minutesValue` and `secondsValue`.
```
Some minor formatting things and removing second person pronoun.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@eskuode I did a second review and had one comment to address.

**Comment by eskuode** [comment]
@SSwiniarski Thanks, I accepted the updates.

**Review by SSwiniarski — APPROVED** [review]
@eskuode Looks good.

**Comment by SSwiniarski** [comment]
@eskuode congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/javascript/dates/setHours

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**
